### DISCUSSION TITLE
Login screen

### DISCUSSION THREAD

**Discussion by chedda61** [DustinBrett/daedalOS#241] [Feature Requests]
Maybe I missed some configuration for this, but I don't get a login screen before showing the desktop.
It's a really cool project, but if someone want's to access this from the internet, it can become quite risky.

I wanna use this mainly to access all of my other web services (zigbee2mqtt, zwave2mqtt, node-red, etc.) - without having to expose and forward all of them - through the browser in daedalOS, which works perfectly...but without a login...

**Comment by DustinBrett** [comment]
There is no login because there is no real users at this point. The entire site/experience is a facsimile of a desktop environment.

Perhaps in the future I will add such a feature, but at the moment it's not integrated in any meaningful way with the system.

As I want the user to get right into the experience, I don't show them a login screen and the desktop is suppose to feel like you've logged in as me, at least the dustinbrett.com version.

**Comment by chedda61** [comment]
Ok, I get that.
However, I noticed that connecting to daedalOS from the internet and accessing one of my not-exposed webservices does actually not work. I guess the connection is not routed through daedalOS but rather established directly through the host browser.
That was the whole point of trying out daedalOS for me unfortunately.
Nonetheless, it's a nice experience and I appreciate the effort. Kudos.

**Comment by DustinBrett** [comment]
Ya currently the browser is just an iframe. I do have plans to add a proxy, but it wouldn't work the way you want until I have a backend aspect, otherwise it's all client side.

**Comment by tmsteph** [comment]
I'm only 1-3 years experienced with react but would be interested in looking into designing a back-end for this system. 

Looking into building an internet based operating system for personal use.

 This seems like a perfect starting point.
 
 As a side, would love to have a full unix linux system running like this. Cloud based OS. 
 
 Thanks for all your work @DustinBrett

**Reply by DustinBrett** [reply]
Thanks! Good luck with the back-end. It's purely client side at the moment as I don't desire to host a back-end and want to do as much as possible client side. But a back-end could indeed add some functionality for someone willing to run it.
